.ng/src/app/models/item.model.ts

```diff
@@ -0,0 +1,176 @@
+
+    export interface BibData {
```

Comment:
@jatjhu I used this [json2ts ](http://www.json2ts.com/) to generate this interface. I think it makes it easier to work with the API because we know all the different properties and types.

Reply:
That makes sense I think. I wouldn't want to try to write it all out manually.